--- museek+-0.2/museeq/chatrooms.cpp.orig	2008-12-03 16:38:36.000000000 +0100
+++ museek+-0.2/museeq/chatrooms.cpp	2017-04-13 22:05:57.664672841 +0200
@@ -55,7 +55,7 @@
 		}
 	}
 
-	ChatRoom* _room = new ChatRoom(room, this, false);
+	ChatRoom* _room = new ChatRoom(room, this);
 	addTab(_room, room);
 	connect(_room, SIGNAL(highlight(int, QWidget*)), this, SIGNAL(highlight(int)));
 	_room->setUsers(r);
--- museek+-0.2/museeq/searches.cpp.orig	2009-02-20 11:17:05.000000000 +0100
+++ museek+-0.2/museeq/searches.cpp	2017-04-13 22:07:49.950210714 +0200
@@ -158,7 +158,7 @@
 
     // Create the tab if it's a wishlist result
 	if ((i == mSearchTabWidget->count()) && museeq->isInWishlist(query)) {
-		Search* s = new Search(query, mSearchTabWidget, false);
+		Search* s = new Search(query, mSearchTabWidget);
 		mSearchTabWidget->addTab(s, query);
 
         // Highlight the search icon so that the user know that there are some new search results
@@ -194,7 +194,7 @@
 		}
 	}
 	if(i == mSearchTabWidget->count()) {
-		Search* s = new Search(q, mSearchTabWidget, false);
+		Search* s = new Search(q, mSearchTabWidget);
 		mSearchTabWidget->addTab(s, q);
 
         // Highlight the search icon so that the user know that there are some new search results
